### PR TITLE
Changed and added commands in docker files

### PR DESCRIPTION
**Pull request by FayKounara**

### CODE REVIEW

**Comment by JohannesLichtenberger** [comment]
Oh, what's this about? I thought it's Keycloak updated somehow

**Comment by FayKounara** [comment]
Hello! So these changes help the container keycloak to not start and exit immediately and when i altered these i was also able to build successfully the build.gradle from the folder sirix-rest-api.

**Comment by JohannesLichtenberger** [comment]
The thing is that the docker compose file in the root dir should build a new image from the sources (afterwards it's pushed to docker hub for use by the rest api bundle and for everyone).

**Comment by FayKounara** [comment]
Do you mean the sirixdb/sirix image?

File: docker-compose.yml [hunk 1]
```diff
@@ -2,30 +2,34 @@ version: '3'
...
     networks:
       - auth-network
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080"]
+      test: "curl --fail  http://localhost:8080 || exit 1"
       interval: 30s
       timeout: 5s
       retries: 10
   server:
-    build: .
```
Comment (JohannesLichtenberger) [inline docker-compose.yml:28]: This should be in, as it's automatically built after successful tests and pushed t docker hub :-)

**Comment by FayKounara** [comment]
Okay so i added this command and executed it again.From the reports that are produced i see this. Package | Tests | Failures | Ignored | Duration | Success rate
-- | -- | -- | -- | -- | --
io.sirix.rest | 66 | 0 | 0 | 19.163s | 100%
So can you please explain what is the problem with the sirix-rest-api because i see all the containers are running smoothly. Maybe i dont understand something. Thank you and sorry for the trouble.

**Comment by JohannesLichtenberger** [comment]
The docker compose file in sirix-rest-api uses the most recent version from docker hub, which is built before via the docker compose file in the root directory of the project. I'm still not really sure what's the problem you're facing. So, using the docker compose script in sirix-rest-api starts keycloak and stops it immediately?

**Comment by FayKounara** [comment]
Sorry for the confusion!  I saw on the issue #711 this,The container doesn't start using the newest image,and also on the issue #707 that the tests fail. I think that these changes might help with these issues.

**Comment by JohannesLichtenberger** [comment]
Oh, #707 was fixed (the keycloak image only exists for quay.io). Idk what's wrong when updating Keycloak to the newest version, though. In any case, if your changes help, we could add the local build again, I guess, regarding the root docker compose file!? Otherwise it wouldn't be updated anymore (and pushed to docker hub).

**Comment by FayKounara** [comment]
If my changes prove effective, I will add the local build option in the root Docker Compose file again.Thank you and sorry for the confusion!

**Comment by JohannesLichtenberger** [comment]
So, you'll update the keycloak version and check?

**Comment by FayKounara** [comment]
Yes i will look into it.

**Comment by JohannesLichtenberger** [comment]
You'd have to update the Keycloak version as well (also add `build .` again, as it's mandatory to do a local image build)

**Comment by JohannesLichtenberger** [comment]
Can welcome this?

**Comment by FayKounara** [comment]
Hello, yes!

**Pull request closed without merging**